.ruby-version

```diff
@@ -0,0 +1 @@
+ruby-2.7.0
```

Comment:
not the same ruby version as the one listed in the GEMFILE

Reply:
I would use 2.5.1 as you have listed in the GEMFILE

Reply:
fix it